BlobStorage/packages.config

```diff
@@ -1,12 +1,23 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Azure.Core" version="1.20.0" targetFramework="net472" />
+  <package id="Azure.Storage.Blobs" version="12.10.0" targetFramework="net472" />
+  <package id="Azure.Storage.Common" version="12.9.0" targetFramework="net472" />
   <package id="Microsoft.Azure.ConfigurationManager" version="4.0.0" targetFramework="net452" />
-  <package id="Microsoft.Azure.KeyVault.Core" version="3.0.3" targetFramework="net452" />
-  <package id="Microsoft.Azure.Storage.Blob" version="10.0.3" targetFramework="net452" />
-  <package id="Microsoft.Azure.Storage.Common" version="10.0.3" targetFramework="net452" />
-  <package id="Microsoft.Data.Edm" version="5.8.4" targetFramework="net452" />
-  <package id="Microsoft.Data.OData" version="5.8.4" targetFramework="net452" />
-  <package id="Microsoft.Data.Services.Client" version="5.8.4" targetFramework="net452" />
+  <package id="Microsoft.Bcl.AsyncInterfaces" version="1.0.0" targetFramework="net472" />
   <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net452" />

```

Comment:
Is there a reason Newtonsoft is still floating around in here? You've added System.Text.Json (what the sdk also uses).